about.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Elite school website</title>
+  <link rel="stylesheet" href="css/about.css">
+
+  <!-------------------------swiper js cocogoose font----------------------------->
+  <link rel="stylesheet" href="https://unpkg.com/swiper@8/swiper-bundle.min.css" />
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&family=Roboto+Flex:opsz,wght@8..144,400;8..144,500;8..144,600&display=swap"
+    rel="stylesheet">
+  <!-------------------------swiper js----------------------------->
+
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="logo">
+        <a href="#home">
+          <img src="img/logo.png" alt="" class="logo">
```

Comment:
- [ ]  Kindly add alt tags to all images, please do this because if your image is not rendered due to network issues or a broken link, users can still see a text that says what the image about, this make your website more accessible